integreat_cms/cms/templates/pois/poi_form.html

```diff
@@ -187,7 +196,43 @@ <h3 class="heading font-bold text-black">
                                  role="alert">
                                 {% trans "Map preview will be available when coordinates are given." %}
                             </div>
-                            <div id="map" class="aspect-video">
+                            <div id="map"
+                                 class="aspect-video"
+                                 data-url="{% url "get_address_from_coordinates" region_slug=request.region.slug %}">
+                            </div>
+                            <div id="drag_and_drop_change" class="hidden">
+                                <div id="new_address_coordinates">
+                                </div>
+                                <label for="drag_and_drop_change" class="secondary !inline">
+                                    {% trans "New address and coordinates found" %}
+                                </label>
+                                <div id="address_found">
+                                </div>
+                                <div id="coordinates_found">
+                                </div>
+                                <input type="radio" id="radio_1" name="radio" value="radio_1"/>
+                                <label for="radio_1">
+                                    {% trans "use address and coodinates of map pin" %}
+                                </label>
+                                <br/>
+                                <input type="radio" id="radio_2" name="radio" value="radio_2"/>
+                                <label for="radio_1">
```

Comment:
copy-paste issue i guess:
`<label for="radio_1">` here suppose to be `<label for="radio_2">`
and the same below for radio_3

Reply:
oh, right 😅 Now fixed!